uaclient/config.py

```diff
@@ -672,8 +679,11 @@ def parse_config(config_path=None):
     for key, value in os.environ.items():
         key = key.lower()
         if key.startswith("ua_"):
-            if "ua_features_" in key:
-                key = key[12:]  # String leading UA_FEATURES_
+            # Strip leading UA_
+            field_name = key[3:]
+            if field_name.startswith("features_"):
```

Comment:
I left in the `ua_features_` support. Let me know if you think we should take this out

Reply:
+1 on this. I think supporting features more often than not is a one-time operation during enable of something "fips" on clouds or potentially enabling a beta service.